src/main/java/org/gridsuite/study/server/service/NetworkModificationTreeService.java

```diff
@@ -778,6 +778,24 @@ public List<UUID> getStudyVoltageInitResultUuids(UUID studyUuid) {
         return resultUuids;
     }
 
+    @Transactional(readOnly = true)
+    public List<UUID> getShortCircuitResultUuids(UUID studyUuid) {
```

Comment:
it's homogeneous with other computations ? The user changes the parameters and it invalidates the results on all nodes ?

Reply:
For all nodes yes but each computation has it's one behaviour :
- if params of LoadFlow changes : SENSI, SA, and dynamique simulation are invalidated on all nodes.
- Sensi provider changes : all nodes are invalidated.
- VoltageInit param changes : invalidate status on all nodes.